### PR TITLE
Eddie/Responsify WhoAreWe

### LINKED ISSUE #77: Responsify WhoWeAre
image

- Make it look good on mobile like the Figma.
- When you are done, on your PR, upload a screenshot of the component on mobile AND laptop/desktop

### PR DESCRIPTION
**Pull request by Ezkron**
image

image

### CODE REVIEW

File: src/components/about/WhoWeAre.jsx [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+    <div className="max-w-7xl mx-auto px-4 py-8 flex flex-col gap-10">
+      <Header title="WHO WE ARE" size="text-4xl" />
+
+      <div className="flex flex-col space-y-10">
+        <p className="font-hk leading-relaxed text-base text-center md:text-left">
+          J&M Products is dedicated to the turnkey manufacturing and marketing
+          of high quality products and services. <br />
+          Through leadership, innovation and continuous improvement we are able
+          to achieve the highest level <br />
+          of customer satisfaction. Our AS9100/ISO9001 approved quality
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx]: remove all these breaks

File: src/components/about/WhoWeAre.jsx [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+          Brochures
+        </p>
+
+        <div className="flex justify-center md:justify-end w-full">
+          <Image
+            src={JMbrochures}
+            alt="Brochures"
+            width={800}
+            height={600}
+            className="w-full max-w-fit"
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx:29]: don't use width and height for Image component.
Use the tailwind classes like `w-1/2` or `w-2/3` whatever you see fit. 
No need for height, just width for tailwind classes. No width and height for Image Component

File: src/components/about/WhoWeAre.jsx [hunk 3]
```diff
@@ -1,7 +1,42 @@
 import React from "react";
+import Image from "next/image";
+import Header from "./Header";
+import JMbrochures from "/public/images/about/JMbrochures.webp";
 
 const WhoWeAre = () => {
-  return <div>WhoWeAre</div>;
+  return (
+    <div className="max-w-7xl mx-auto px-4 py-8 flex flex-col gap-10">
+      <Header title="WHO WE ARE" size="text-4xl" />
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx]: remove the `size` prop, since we removed it

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by Ezkron** [comment]
image

image

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**